core/renderers/zelos/info.js

```diff
@@ -278,8 +278,9 @@ Blockly.zelos.RenderInfo.prototype.addInput_ = function(input, activeRow) {
 Blockly.zelos.RenderInfo.prototype.addAlignmentPadding_ = function(row,
     missingSpace) {
   if (row.rightAlignedDummyInput) {
-    var alignmentDivider;
-    for (var i = 0, elem; (elem = row.elements[i]); i++) {
+    let alignmentDivider;
+    let i = 0, elem;
+    for (; (elem = row.elements[i]); i++) {
```

Comment:
I think for now we're keeping these declarations inside the for loop.

Reply:
Oh yes. I forgot to check for this. my IDE does weird things with for loops when I ask it to convert var to let/const. 
I'll fix all these.